src/wallet/rpcwallet.cpp

```diff
@@ -3533,11 +3534,15 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
     UniValue contextInfo = o;
 
     // Contextual transaction we will build on
+    int nextBlockHeight = chainActive.Height() + 1;
     CMutableTransaction contextualTx = CreateNewContextualCMutableTransaction(
-        Params().GetConsensus(), chainActive.Height() + 1);
+        Params().GetConsensus(), nextBlockHeight);
```

Comment:
This is going to have a merge conflict with #2978. I'll leave it to you two to decide who blinks :smile: